initCobraToolbox.m

```diff
@@ -171,7 +171,7 @@ function initCobraToolbox()
 %If the Matlab version is prior to Matlab 2016a, we have to use the TC.xml
 %structure to provide autocompletion, otherwise we can use the json
 %definition.
-if MatlabYear < 2015
+if MatlabYear < 2016

```

Comment:
Sorry, I got that wrong. I wanted to say `> 2015` to inform the user that `tabcomplete` works. 😄